setup.py

```diff
@@ -53,9 +53,10 @@
     "hf_transfer>=0.1.4",
     "huggingface-hub[cli]>=0.19.2,<1.0",
     "isort>=5.12.0",
+    "langdetect",  # Needed for LightEval's extended tasks
     "latex2sympy2_extended>=1.0.6",
     "liger_kernel==0.5.2",
-    "lighteval @ git+https://github.com/huggingface/lighteval.git@86f62259f105ae164f655e0b91c92a823a742724#egg=lighteval[math]",
+    "lighteval @ git+https://github.com/huggingface/lighteval.git@e89a9876ada1da511df0708955a63012739222b5",
```

Comment:
Important fix for generation parameter parsing: https://github.com/huggingface/lighteval/pull/578